### ISSUE TITLE
Capture display language in telemetry

### ISSUE THREAD

**Issue by jakobpn** [OfficeDev/script-lab#476]
As we are enabling translation of Script Lab into other languages than English, it would be good to capture the display language in the custom events that is submitted to App Insights.

**Comment by Zlatkovsky** [comment]
Agreed.  Sounds like a good one for @cshashwat to do, to get some experience into App Insights :-)
